samples/BasicXrApp/OpenXrProgram.cpp

```diff
@@ -601,7 +611,8 @@ namespace {
 
         XrSpaceHandle m_sceneSpace;
 
-        xr::XrHandle<XrSpatialAnchorMSFT, xrDestroySpatialAnchorMSFT> m_spatialAnchor;
+        typedef xr::XrHandle<XrSpatialAnchorMSFT, xrDestroySpatialAnchorMSFT> SpatialAnchorHandle;
```

Comment:
This typedef should just be in XrHandle.h, since that helper sits in the same branch with the openxr.h that supports the `XR_MSFT_spatial_anchor` extension.